<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CorrelationBayesianForm</class>
 <widget class="QWidget" name="CorrelationBayesianForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>876</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="topWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="3" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" rowspan="4">
       <widget class="BoundListView" name="variables">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="AssignButton" name="assignButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="AvailableFieldsListView" name="availableVariables">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="optionsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QGridLayout" name="gridLayout_11">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="GroupBox" name="groupBox_3">
           <property name="title">
            <string>Correlation Coefficients</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0">
             <widget class="BoundCheckBox" name="spearman">
              <property name="text">
               <string>Spearman</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="BoundCheckBox" name="pearson">
              <property name="text">
               <string>Pearson's rho</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="BoundCheckBox" name="kendallsTauB">
              <property name="text">
               <string>Kendall's tau-b</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_7" native="true">
           <layout class="QGridLayout" name="gridLayout_5">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <item row="4" column="1">
             <widget class="QWidget" name="ciValueContainer" native="true">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QWidget" name="widget_8" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>%</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="BoundTextBox" name="ciValue">
                 <property name="maximumSize">
                  <size>
                   <width>60</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Interval</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>0</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="BoundCheckBox" name="reportBayesFactors">
              <property name="text">
               <string>Report Bayes factors</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="BoundCheckBox" name="credibleInterval">
              <property name="text">
               <string>Credible intervals</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="BoundCheckBox" name="flagSupported">
              <property name="text">
               <string>Flag supported correlations</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BoundGroupBox" name="hypothesis">
           <property name="title">
            <string>Hypothesis</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QRadioButton" name="_1correlated">
              <property name="text">
               <string>Correlated</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="_3correlatedNegatively">
              <property name="text">
               <string>Correlated negatively</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="_2correlatedPositively">
              <property name="text">
               <string>Correlated positively</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="BoundGroupBox" name="bayesFactorType">
           <property name="title">
            <string>Bayes Factor</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_1">
              <property name="text">
               <string>BF₁₀</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_2">
              <property name="text">
               <string>BF₀₁</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton_3">
              <property name="text">
               <string>Log(BF₁₀)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="GroupBox" name="correlationMatrixContainer">
           <property name="title">
            <string>Plots</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="1" column="0" rowspan="2">
             <widget class="QWidget" name="widget_9" native="true">
              <property name="maximumSize">
               <size>
                <width>12</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="BoundCheckBox" name="plotDensitiesForVariables">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Densities for variables</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="BoundCheckBox" name="plotPosteriors">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Posteriors under H₁</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="BoundCheckBox" name="plotCorrelationMatrix">
              <property name="text">
               <string>Correlation matrix</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="groupPrior" native="true">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="1">
             <widget class="BoundTextBox" name="priorWidth">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Stretched beta prior width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="verticalWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="ExpanderButton" name="buttonOptions">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Options</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BoundGroupBox" name="missingValues">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Missing Values</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QRadioButton" name="_1excludePairwise">
           <property name="text">
            <string>Exclude cases pairwise</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="_2excludeListwise">
           <property name="text">
            <string>Exclude cases listwise</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/groupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundTextBox</class>
   <extends>QLineEdit</extends>
   <header>widgets/boundtextbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundGroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/boundgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>plotCorrelationMatrix</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotDensitiesForVariables</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plotCorrelationMatrix</sender>
   <signal>toggled(bool)</signal>
   <receiver>plotPosteriors</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>credibleInterval</sender>
   <signal>toggled(bool)</signal>
   <receiver>ciValueContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
